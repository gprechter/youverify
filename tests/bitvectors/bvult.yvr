a,b,c,d: BV[8]
r1,r2,r3,r4: BOOL
a = BV{4, 8}
b = BV{8, 8}
c = BV{1, 8}
d = BV{16, 8}

call r1 = bvult(a, b)
call r2 = bvult(a, c)
call r3 = bvult(d, b)
call r4 = bvult(BV{0, 8}, c)

