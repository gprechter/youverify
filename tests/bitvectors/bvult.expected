{
    "a": 4,
    "b": 8,
    "c": 1,
    "d": 16,
    "r1": true,
    "r2": false,
    "r3": false,
    "r4": true
}